src/main/java/com/solvd/carsharing/command/util/CommandUtilsImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.solvd.carsharing.command.util;
+
+import com.solvd.carsharing.kafka.property.KfProperties;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+@Service
+@RequiredArgsConstructor
+public class CommandUtilsImpl implements CommandUtils {
```

Comment:
@vpetrovaa if it's utility class then it should be final class without a chance to construct it

if it's object that can perform some functionality -> should be named properly

Reply:
@h1alexbel, ok, this method is connected with access to another microservice. Is RentalService the correct name? So this method changes the status of rental https://github.com/vpetrovaa/carsharing/pull/4/commits/1d6b4e55308e74d734f7618e6bad13d1633ef852